### ISSUE TITLE
Not compatible with current managed Contao version 4.11.5 (with Symfony 5)

### ISSUE THREAD

**Issue by Baloobear** [OMOSde/contao-om-imagine#10]
Hello,

your current version (v1.1.0) seems not to be compatible with the Symfony 5 framwork because of the dependency with the wa72/htmlpagedom lib of v1.x.

```
/usr/src/app$ composer require omosde/contao-om-imagine
Using version ^1.1 for omosde/contao-om-imagine
./composer.json has been updated
Running composer update omosde/contao-om-imagine
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - wa72/htmlpagedom v1.3.0 requires symfony/dom-crawler ^2.3 <2.8 | ^2.8.3 | ^3.0.1 -> found symfony/dom-crawler[v2.3.0, ..., v2.8.52, v3.0.1, ..., v3.4.47] but the package is fixed to v5.2.10 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - wa72/htmlpagedom v1.3.1 requires symfony/dom-crawler ^2.3 <2.8 | ^2.8.3 | ^3.0.1 | ^4 -> found symfony/dom-crawler[v2.3.0, ..., v2.8.52, v3.0.1, ..., v3.4.47, v4.0.0, ..., v4.4.27] but the package is fixed to v5.2.10 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - wa72/htmlpagedom[v1.3.2, ..., v1.4.2] require symfony/dom-crawler ^2.3 <2.8 | ^2.8.3 | ^3.0.1 | ^4 <4.3 -> found symfony/dom-crawler[v2.3.0, ..., v2.8.52, v3.0.1, ..., v3.4.47, v4.0.0, ..., v4.2.12] but the package is fixed to v5.2.10 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - omosde/contao-om-imagine 1.1.0 requires wa72/htmlpagedom ^1.3 -> satisfiable by wa72/htmlpagedom[v1.3.0, ..., v1.4.2].
    - Root composer.json requires omosde/contao-om-imagine ^1.1 -> satisfiable by omosde/contao-om-imagine[1.1.0].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

But the mentioned lib has already a version 2 (v2.0.1) which seems to be compatible with the Symfony 5 framework, but might have some corresponding breaking changes. So maybe there's a chance that this lib can be updated to be compatible with the Symfony 5 framework?

I'll appreciate this.

Or do you know any alternative Contao extension to automatically watermark images? 😉

Thanks and kind regards,
Dennis

**Comment by OMOSde** [comment]
Hello Dennis,

I would be happy to take a look at the problem over the weekend. Should not be very complex.

Many greetings
René

**Comment by OMOSde** [comment]
Fixed in 5912cf2797c9b2e12a6bfd069029a9f5a028a46d
